sync_diff_inspector/diff.go

```diff
@@ -355,27 +362,44 @@ func (df *Diff) Equal() (err error) {
 
 	for _, schema := range df.tables {
 		for _, table := range schema {
+			var tidbStatsSource *diff.TableInstance
+
 			sourceTables := make([]*diff.TableInstance, 0, len(table.SourceTables))
 			for _, sourceTable := range table.SourceTables {
-				sourceTables = append(sourceTables, &diff.TableInstance{
+				sourceTableInstance := &diff.TableInstance{
 					Conn:   df.sourceDBs[sourceTable.InstanceID].Conn,
 					Schema: sourceTable.Schema,
 					Table:  sourceTable.Table,
-				})
+				}
+				sourceTables = append(sourceTables, sourceTableInstance)
+
+				if sourceTable.InstanceID == df.tidbInstanceID {
+					tidbStatsSource = sourceTableInstance
+				}
+			}
+
+			targetTableInstance := &diff.TableInstance{
+				Conn:   df.targetDB.Conn,
+				Schema: table.Schema,
+				Table:  table.Table,
+			}
+
+			if df.targetDB.InstanceID == df.tidbInstanceID {
```

Comment:
add `instance-id` field for `[target-db]` in *config.toml* as an exmaple?
in addition, is it needing to check whether `instance-id` duplicated  between source and target DBs?